Mega Man/Forms/MainForm.cs

```diff
@@ -68,6 +68,20 @@ private void DefaultScreen()
 
             ResizeScreen();
         }
+        
+        protected override void OnResize(EventArgs e)
+        {

```

Comment:
Looks good, just make sure doing other types of resizes, and therefore calling Start() repeatedly, doesn't cause any issues.